pkgs/top-level/all-packages.nix

```
@@ -19397,6 +19397,10 @@ in
 
   musl = callPackage ../os-specific/linux/musl { };
 
+  musl-fts = callPackage ../os-specific/linux/musl-fts { };
+  musl-obstack = callPackage ../os-specific/linux/musl-obstack { };
+  obstack = if stdenv.hostPlatform.isMusl then musl-obstack else null;
```

```suggestion
  musl-fts = callPackage ../os-specific/linux/musl-fts { };

  musl-obstack = callPackage ../os-specific/linux/musl-obstack { };
```

We can't default a package to null conditionally because it usually breaks eval in some form. I would suggest we just input musl-obstack into elfutilsdirectly.

---

SGTM, did that copying the way BSD FTS was done but i agree it's kind of specious. I'll ditch it.